### ISSUE TITLE
FileNotFoundError: No such file or no access: 'sub-01_t1w_preproc.nii.gz'

### ISSUE THREAD

**Issue by TamerGezici** [miykael/nipype_tutorial#165]
I am running the tutorial dockerfile and I ran into an issue like this at the example_preprocessing notebook.

While executing cell:

```
plotting.plot_epi(
    '/data/ds000114/derivatives/fmriprep/sub-01/anat/sub-01_t1w_preproc.nii.gz',
    title="T1", display_mode='ortho', annotate=False, draw_cross=False, cmap='gray');
```

`FileNotFoundError: No such file or no access: '/data/ds000114/derivatives/fmriprep/sub-01/anat/sub-01_t1w_preproc.nii.gz'`

The file is there, I checked.

**Comment by JohannesWiesner** [comment]
Did you ran `introduction.ipynb` before that? I think I stumpled upon the same `FileNotFoundError` but it's important to run this script before

**Comment by TamerGezici** [comment]
> Did you ran `introduction.ipynb` before that? I think I stumpled upon the same `FileNotFoundError` but it's important to run this script before

Hi. Thanks a lot for your answer. I actually haven't so this could be why.

There is no single notebook called "introduction" but several with the prefix "introduction" is it the one called "introduction_dataset" that I should be running?

**Comment by JohannesWiesner** [comment]
Yes, you're right, I meant `introduction_dataset.ipynb`. Put please take this with a grain of salt I could also be wrong here (also trying out this tutorial for the first time).  From the [homepage](https://miykael.github.io/nipype_tutorial/) it looks like one could directly start with `example_preprocessing.ipynb` but as you read the first lines of `example_preprocessing.ipynb`one already sees that `introduction_dataset.ipynb` has to be run before that (took me a while to find that out):

> Preparation
> Before we can start with anything we first need to download the data (the other 9 subjects in the dataset). This can be done very quickly with the following datalad command.
> 
> Note: This might take a while, as datalad needs to download ~700MB of data
> 
> %%bash
> datalad get -J 4 /data/ds000114/derivatives/fmriprep/sub-*/anat/*preproc.nii.gz \
>                 /data/ds000114/sub-*/ses-test/func/*fingerfootlips*
